src/main/scala/dsptools/VerboseDspTester.scala

```diff
@@ -0,0 +1,303 @@
+// See LICENSE for license details.
+
+package dsptools
+
+import breeze.math.Complex
+import dsptools.numbers.{DspComplex, DspReal, Real}
+import chisel3._
+import chisel3.iotesters.TestersCompatibility
+import dsptools.Utilities.bigIntBitsToDouble
+import chisel3.internal.InstanceId
+
+import scala.util.DynamicVariable
+
+class VerboseDspTester[T <: Module](dut: T,
+    base: Int = 16,
+    logFile: Option[java.io.File] = None) extends DspTester(dut, base, logFile) with VerilogTbDump {
+
+  val updatableBase = new DynamicVariable[Int](_base)
+  private def dispBase = updatableBase.value
+
+  // Verbosity @ DSP level (different from chisel-testers verbosity)
+  val updatableDSPVerbose = new DynamicVariable[Boolean](dsptestersOpt.isVerbose)
+  private def dispDSP = updatableDSPVerbose.value
+
+  // Verbosity @ normal peek/poke level (chisel-testers verbosity) -- for additionally displaying as bits
+  val updatableBitVerbose = new DynamicVariable[Boolean](iotestersOM.testerOptions.isVerbose)
+  private def dispBits = updatableBitVerbose.value
+
+  val fixTolLSBs = new DynamicVariable[Int](dsptestersOpt.fixTolLSBs)
+  private def fixTol = fixTolLSBs.value
+
+  val realTolDecPts = new DynamicVariable[Int](dsptestersOpt.realTolDecPts)
+  private def realTol = realTolDecPts.value
+
+  // Tester starts with reset (but don't count cycles during reset)
+  super.reset(5)
+
+  private def getName(signal: InstanceId): String = {
+    s"${signal.parentPathName}.${TestersCompatibility.validName(signal.instanceName)}"
+  }
+
+  private def validRangeTest(signal: Bits, value: BigInt) {
+    val len = value.bitLength
+    val neededLen = if (isSigned(signal)) len + 1 else len
+    if (neededLen > signal.getWidth) 
+      throw DspException(s"Poke/Expect value of ${getName(signal)} is not in node range")
+  }
+
+  // TODO: Get rid of duplication in chisel-testers
+  private def signConvert(signal: Bits, bigInt: BigInt): BigInt = {
+    val width = signal.getWidth
+    val signFix = {
+      if(bigInt.bitLength >= width) - ((BigInt(1) << width) - bigInt)
+      else bigInt
+    }
+    if (isSigned(signal)) signFix else bigInt
+  }
+
+  ///////////////// OVERRIDE UNDERLYING FUNCTIONS FROM PEEK POKE TESTER /////////////////
+
+  override def step(n: Int) {
+    if (dispDSP) logger println s"STEP ${n}x -> ${t+n}"
+    stepPrint(n)
+    backend.step(n)
```

Comment:
The problem is I can't change _verbose in PeekPokeTester easily. Unless I mess around with the DynamicVariable inside optionsManager.testerOptions.isVerbose. Even then, I have an alternative purpose for that particular isVerbose (i.e. print out everything bit-wise, rather than as Double for dspPeeks, etc.). The backend versions of stuff lets me directly pass in verbose as an argument. If someone wants to provide an alternative, that's fine... but again, I'm just really going around restrictions of chisel-testers...